mmcv/runner/hooks/evaluation.py

```diff
@@ -367,8 +391,13 @@ def _do_evaluate(self, runner):
         if tmpdir is None:
             tmpdir = osp.join(runner.work_dir, '.eval_hook')
 
-        from mmcv.engine import multi_gpu_test
-        results = multi_gpu_test(
+        if self.test_fn is None:
```

Comment:
How about we do this once during initialization rather than import the function every epoch?